tests/python/unit_tests/test_optimizers/test_pybernyop.py

```diff
@@ -27,19 +35,39 @@ def test_optimize_pyberny(self):
         mm = pp.ModuleManager()
         nwchemex.load_modules(mm)
         structurefinder.load_modules(mm)
-        mm.change_input("NWChem : SCF", "basis set", "sto-3g")
-        mm.change_input("NWChem : SCF Gradient", "basis set", "sto-3g")
-        mm.change_submod("PyBerny", "Gradient", "NWChem : SCF Gradient")
-        mm.change_submod("PyBerny", "Energy", "NWChem : SCF")
-        mm.change_submod("Pyberny", "StringConv",
-                         "ChemicalSystem via QCElemental")
-        egy = mm.run_as(TotalEnergy(), "PyBerny",
-                        chemist.ChemicalSystem(self.mol))
-        print("Energy = " + str(egy))
-        print(np.array(egy).item())
-        self.assertAlmostEqual(np.array(egy).item(), -1.117505879316, 10)
+
+        pyberny_mod = mm.at("PyBerny")
+        nwchem_scf_mod = mm.at("NWChem : SCF")
+        nwchem_grad_mod = mm.at("NWChem : SCF Gradient")
+        string_conv_mod = mm.at("ChemicalSystem via QCElemental")
+
+        nwchem_scf_mod.change_input("basis set", "sto-3g")
+        nwchem_grad_mod.change_input("basis set", "sto-3g")
+        pyberny_mod.change_submod("Energy", nwchem_scf_mod)
+        pyberny_mod.change_submod("Gradient", nwchem_grad_mod)
+        pyberny_mod.change_submod("StringConv", string_conv_mod)
+
+        egy = pyberny_mod.run_as(TotalEnergyNuclearOptimization(), self.sys,
```

Comment:
```suggestion
        energy, points = pyberny_mod.run_as(TotalEnergyNuclearOptimization(), self.sys,
```
I think this will work and it's less confusing than referring to both as `egy`